### ISSUE TITLE
fails to apply f-droid patch while building for blueline

### ISSUE THREAD

**Issue by pf31** [hashbang/os#16]
Building for blueline. Cloned the repo and issued:
`make DEVICE=blueline keys`
`make DEVICE=blueline clean build release`. 

It fails with
> Applying patch: /opt/android/patches/platform/add-fdroid.patch
patching file build/make/target/product/core.mk
Hunk #1 succeeded at 39 (offset 2 lines).
patching file packages/apps/F-Droid/Android.mk
Hunk #1 FAILED at 10.
1 out of 1 hunk FAILED -- saving rejects to file packages/apps/F-Droid/Android.mk.rej
patching file packages/apps/F-Droid/app/local.properties
patching file packages/apps/F-Droid/local.properties
Makefile:49: recipe for target 'build' failed
make: *** [build] Error 1

F-Droid's Android.mk is not as the patch expects it to be.
edit: add output
```
build@android:~$ cat base/packages/apps/F-Droid/Android.mk.rej
--- packages/apps/F-Droid/Android.mk
+++ packages/apps/F-Droid/Android.mk
@@ -10,13 +10,13 @@ fdroid_root  := $(LOCAL_PATH)
 fdroid_dir   := app
 fdroid_out   := $(PWD)/$(OUT_DIR)/target/common/obj/APPS/$(LOCAL_MODULE)_intermediates
 fdroid_build := $(fdroid_root)/$(fdroid_dir)/build
-fdroid_apk   := build/outputs/apk/$(fdroid_dir)-release-unsigned.apk
+fdroid_apk   := build/outputs/apk/full/release/app-full-release-unsigned.apk

 $(fdroid_root)/$(fdroid_dir)/$(fdroid_apk):
        rm -Rf $(fdroid_build)
        mkdir -p $(fdroid_out)
        ln -sf $(fdroid_out) $(fdroid_build)
-       cd $(fdroid_root)/$(fdroid_dir) && gradle assembleRelease
+       cd $(fdroid_root)/$(fdroid_dir) && ../gradlew assembleRelease

 LOCAL_CERTIFICATE := platform
 LOCAL_SRC_FILES := $(fdroid_dir)/$(fdroid_apk)
```

Android.mk looks like:
```
build@android:~$ cat base/packages/apps/F-Droid/Android.mk
LOCAL_PATH:= $(call my-dir)

include $(CLEAR_VARS)

LOCAL_MODULE := F-Droid
LOCAL_MODULE_TAGS := optional
LOCAL_PACKAGE_NAME := F-Droid

fdroid_root  := $(LOCAL_PATH)
fdroid_dir   := app
fdroid_out   := $(PWD)/$(OUT_DIR)/target/common/obj/APPS/$(LOCAL_MODULE)_intermediates
fdroid_build := $(fdroid_root)/$(fdroid_dir)/build
fdroid_apk   := build/outputs/apk/full/release/$(fdroid_dir)-full-release-unsigned.apk

$(fdroid_root)/$(fdroid_dir)/$(fdroid_apk):
        rm -Rf $(fdroid_build)
        mkdir -p $(fdroid_out)
        ln -sf $(fdroid_out) $(fdroid_build)
        cd $(fdroid_root)/$(fdroid_dir) && gradle assembleRelease

LOCAL_CERTIFICATE := platform
LOCAL_SRC_FILES := $(fdroid_dir)/$(fdroid_apk)
LOCAL_MODULE_CLASS := APPS
LOCAL_MODULE_SUFFIX := $(COMMON_ANDROID_PACKAGE_SUFFIX)

include $(BUILD_PREBUILT)
```

**Comment by daurnimator** [comment]
I get the same failure.

**Comment by RyanSquared** [comment]
Edited by @pf31 as of about right now. GitHub doesn't (I think) notify people, so try to not edit things in the future with added content without bumping.

**Comment by pf31** [comment]
Ok. Added lines to the failure output.

**Comment by lrvick** [comment]
fixed in master (a441510f922d5158e854a67f9f3b121a64d1df47)
